nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestCommentsMigrator.java

```diff
@@ -488,8 +488,14 @@ protected void createCommentsAsProperty(boolean commentsWithoutParent) {
             NuxeoPrincipal principal = session.getPrincipal();
             for (int i = 0; i < NB_COMMENTS_BY_FILE; i++) {
                 DocumentModel comment = session.createDocumentModel(null, "comment_" + i, COMMENT_DOC_TYPE);
+                comment.setPropertyValue(COMMENT_PARENT_ID, fileWithCommentWithoutParent.getId());
```

Comment:
Shouldn't be done by the service?

Reply:
i thought the same think. I think we should (new refacto....) add some validation comment manager if the parent is missed. actually this is done on `CommentableDocumentAdapter` i mean the code part that add the parent. it why i added it here manually